### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by r-kyatham**
Move Dashboard setup user guide into README

### CODE REVIEW

File: examples/cost-optimization-dashboard/README.md [hunk 1]
```diff
@@ -1,5 +1,176 @@
 # Cost Optimization Dashboard
 
-The repo contains SQL scripts for analyzing Google Cloud Billing,
-Recommendations data. The queries help in building Cost Optimization Dashboard.
+This repo contains SQL scripts for analyzing GCP Billing, Recommendations data and also a guide to setup the Cost Optimization dashboard.
 For sample dashboard [see here](https://datastudio.google.com/c/u/0/reporting/0c341178-d776-4617-b887-db981f081a76/page/WXzW).
```
Comment (boredabdel) [inline examples/cost-optimization-dashboard/README.md:4]: This link gives me a 

```
Can't access report
Your current account abdelfettah@google.com can't access this report, or the report doesn't exist.
```

is this normal ?

**Review by boredabdel — COMMENTED** [review]
Only one comment to address and this is ready to be merged

File: examples/cost-optimization-dashboard/README.md (hunk 1 above)
Comment (r-kyatham) [reply examples/cost-optimization-dashboard/README.md]: Thanks for the review and good catch. I have update the link.

**Review by boredabdel — APPROVED** [review]
LGTM

**Pull request merged**